src/svc/invocable-scripts/schemas/ScriptUpdateRequest.yml

```diff
@@ -1,9 +1,8 @@
 type: object
 properties:
-  name:
```

Comment:
`name` is removed from the schema?

Reply:
that is correct. I tested it a few times and name cannot be updated.

Reply:
Sounds like this might change again soon (per Slack thread).